omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/FulfillerStatusResource2_2WebTest.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+import org.openmrs.Order;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.v1_0.RestTestConstants2_2;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+
+
+public class FulfillerStatusResource2_2WebTest extends MainResourceControllerTest {
+	
+	@Before
+	public void setUp() throws Exception {
+		Order order = Context.getOrderService().getOrderByUuid(RestTestConstants2_2.ORDER_UUID).copy();
+		order.setUuid(RestTestConstants2_2.ORDER_UUID_Web);
+		order.setOrderId(null);
+		Context.getOrderService().saveOrder(order, null);
+	}
+	
+	@Test
+	public void shouldCreateNew() throws Exception {
```

Comment:
This test should be named along the lines of what it actually does, i.e. set fulfiller details.